tracer/test/Datadog.Trace.Security.Unit.Tests/WafCompatibilityTests.cs

```diff
@@ -15,28 +15,17 @@ namespace Datadog.Trace.Security.Unit.Tests
     [Collection(nameof(SecuritySequentialTests))]
     public class WafCompatibilityTests
     {
-        [SkippableFact]
-        public void ShouldNotInitializeWithExportsMissing()
+        [SkippableTheory]
+        [InlineData("1.3.0")]
+        [InlineData("1.10.0")]
+        public void ShouldNotInitializeWithExportsMissing(string version)
         {
             // for some reason, these tests cause a "DDWAF_ERROR": [ddwaf_run]interface.cpp(206): std::bad_alloc on mac when run with others
             SkipOn.Platform(SkipOn.PlatformValue.MacOs);
-            var libraryInitializationResult = WafLibraryInvoker.Initialize("1.3.0");
+            var libraryInitializationResult = WafLibraryInvoker.Initialize(version);
             libraryInitializationResult.ExportErrorHappened.Should().BeTrue();
             libraryInitializationResult.Success.Should().BeFalse();
             libraryInitializationResult.WafLibraryInvoker.Should().BeNull();
         }
-
-        [SkippableFact]
-        public void ShouldNotInitializeWithDiagnosticsMissing()
-        {
```

Comment:
has to go as now 1.10 doesnt have the correct exports, so we dont initialize the library anymore